src/quo/components/wallet/transaction_summary/schema.cljs

```diff
@@ -0,0 +1,23 @@
+(ns quo.components.wallet.transaction-summary.schema
```

Comment:
Why make a new `ns` ? Is there any guideline I missed? Or did we get circular dependancies ?

Reply:
usually we would put the schema in a separate ns if it was getting big, but there's no guideline for this and this one seems fine to have together with the view

Reply:
I don't get this. Can you please explain more? @shivekkhurana

Reply:
I'm sorry, I was just putting everything in view.cljs
and it was approved. I didn't notice you had extracted everything in schema namespace.
I'll do the same for batch-2